### PR TITLE
feat: add health checks for storage.cnrm.cloud.google.com/StorageBucketAccessControl

### PR DESCRIPTION
**Pull request by vgelot**
Add feature for storage.cnrm.cloud.google.com StorageBucketAccessControl

### CODE REVIEW

File: ui/src/app/settings/components/repos-list/repos-list.tsx [hunk 1]
```diff
@@ -266,9 +266,17 @@ export class ReposList extends React.Component<
                                                         <i className={'icon argo-icon-' + (repo.type || 'git')} />
                                                     </div>
                                                     <div className='columns small-1'>{repo.type || 'git'}</div>
-                                                    <div className='columns small-2'>{repo.name}</div>
+                                                    <div className='columns small-2'>
+                                                        <Tooltip content={repo.name}>
+                                                            <span>{repo.name}</span>
+                                                        </Tooltip>
+                                                    </div>
```
Comment (crenshaw-dev) [inline ui/src/app/settings/components/repos-list/repos-list.tsx:273]: @vgelot I don't think I understand this change... why have a tooltip that just repeats the text?

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]
@vgelot if you want to split the tooltip part into a new PR, I'll happily merge the health check!

File: ui/src/app/settings/components/repos-list/repos-list.tsx (hunk 1 above)
Comment (vgelot) [reply ui/src/app/settings/components/repos-list/repos-list.tsx]: Sorry, it was a false manip during my rebase
I just rebased

**Review by crenshaw-dev — APPROVED** [review]
lgtm, thanks @vgelot!

**Pull request merged**